### PR TITLE
Add test that presto is started as presto user

### PR DESCRIPTION
**Pull request by cawallin**
* Make sure that the user of server's pid is presto
* change run_prestoadmin_script to run_script_from_prestoadmin_dir and
  extend to be able to run on other hosts than master

Testing: make lint, ran test, sandbox with all tests and new RPM

### CODE REVIEW

**Comment by cawallin** [comment]
This patch will need to be added to a special release branch, because it only works with 115t (and the fix in Presto will take a bit to be merged).

**Comment by cawallin** [comment]
@rschlussel @anusudarsan please review

File: tests/product/test_control.py [hunk 1]
```diff
@@ -268,6 +268,19 @@ def test_start_restart_config_file_error(self):
         self.assert_simple_server_restart(expected_restart,
                                           expected_stop=expected_stop)
 
+    def test_started_with_presto_user(self):
+        # note, will only work with 0.115t RPM
+        self.setup_cluster('presto')
```
Comment (anusudarsan) [inline tests/product/test_control.py]: can we extract 'presto' to a const?

**Comment by anusudarsan** [comment]
looks good

File: tests/product/base_product_case.py [hunk 2]
```diff
@@ -348,17 +348,21 @@ def run_prestoadmin(self, command, raise_error=True, cluster=None):
             raise_error=raise_error
         )
 
-    def run_prestoadmin_script(self, script_contents, **kwargs):
+    def run_script_from_prestoadmin_dir(self, script_contents, host='',
```
Comment (rschlussel) [inline tests/product/base_product_case.py]: why don't you give host a default value of self.cluster.master?

Comment (rschlussel) [inline tests/product/base_product_case.py]: oh can it not get self as a value?

**Comment by rschlussel** [comment]
looks good

File: tests/product/base_product_case.py (hunk 2 above)
Comment (cawallin) [inline tests/product/base_product_case.py]: yeah, it can't have self in the default

File: tests/product/test_control.py (hunk 1 above)
Comment (cawallin) [inline tests/product/test_control.py]: good idea, I'll do this in a different commit though

**Pull request closed without merging**